src/mz390.java

```diff
@@ -2484,6 +2486,10 @@ private void load_get_mac_file_line(){
 			while (retry){
 				retry = false;  
 				tz390.systerm_io++;
+				tot_get_mac++; // issue #343 count bal lines 
+				if (tot_get_mac > tz390.opt_maxline){
+				   abort_error(270,"maxium bal lines exceeded");
```

Comment:
Typo. "maxium" should be "maximum". Also, error 270 is also issued just below line  15280; message there is "maximum ainsert source lines MAXLINE exceeded". Should the message here at line 2491 be rewritten to match that format?